applications/adcirc-frontera/app.json

```diff
@@ -0,0 +1,92 @@
+{
+    "id": "adcirc",
```

Comment:
Can you add the paths to the subdirs of each `app.json` to `initialize_tenant.py`? See figuregen for reference

Reply:
https://github.com/TACC/WMA-Tapis-Templates/blob/561174af3bba776a26c99d28c234fcbbfcd9cff6/initialize_tenant.py#L157